api/index.js

```diff
@@ -31,6 +31,25 @@ function TaskAPIFactory (PouchDB) {
         taskStore.remove(taskDoc)
       }
 
+      api.progress = function (taskDoc, options) {
+        if (!taskDoc) throw new Error('Task Error requires taskdoc as first argument')
+
+        // merge options with defaults
+        let progressState = Object.assign({
```

Comment:
yes, I missed that. We can use lodash methods, as we use them at other places in Hoodie already